pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/QueryQuotaClusterIntegrationTest.java

```diff
@@ -333,11 +389,15 @@ private void runQueries(int qps, boolean shouldFail, String applicationName) {
   private static volatile String _quotaSource;
 
   private void verifyQuotaUpdate(double quotaQps) {
+    verifyQuotaUpdateWithTableName(quotaQps, getTableName() + "_OFFLINE");
+  }
+
+  private void verifyQuotaUpdateWithTableName(double quotaQps, String tableName) {
     try {
       TestUtils.waitForCondition(aVoid -> {
         try {
           double tableQuota = Double.parseDouble(sendGetRequest(
-              "http://" + _brokerHostPort + "/debug/tables/queryQuota/" + getTableName() + "_OFFLINE"));
+              "http://" + _brokerHostPort + "/debug/tables/queryQuota/" + tableName));
```

Comment:
Logical table query quota is also checked using the same API ?

Reply:
Yes. It is a table name to query quota mapping so it works.
Link: https://github.com/apache/pinot/blob/61de01bd529fd37b26a33ad4dc113e1d3ce0aa76/pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java#L633